src/os/bluestore/BlueStore.cc

```diff
@@ -12083,7 +12083,9 @@ int BlueStore::omap_get_values(
       r = -ENOENT;
       goto out;
     }
-    iter->upper_bound(*start_after);
```

Comment:
not sure if related to this PR?

Reply:
Well, it is related to this PR only accidentally.
It is an obvious bug but was not exposed until this PR.